src/sentry/options/defaults.py

```diff
@@ -173,6 +173,9 @@
 # Skip saving an event to postgres
 register("store.skip-pg-save", default=True, flags=FLAG_PRIORITIZE_DISK)
 
+# Use Django event
+register("store.use-django-event", default=False, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
We'll run tests with the new behavior (false), but set to True in prod before deploying code that uses this.